servers/rendering/renderer_rd/shaders/scene_data_inc.glsl

```diff
@@ -0,0 +1,74 @@
+// Scene data stores all our 3D rendering globals for a frame such as our matrices
+// where this information is independent of the different RD implementations.
+// This enables us to use this UBO in our main scene render shaders but also in
+// effects that need access to this data.
+
+// We define highp and mediump for the benefit of our mobile renderer
+// esp mediump may have adverse effects on the clustered renderer
+// if so we need to look into a way to switch this.
+// That said, I'm not sure highp and mediump change the actual UBO...
```

Comment:
Given our conversation yesterday, I would just remove this comment.

Reply:
agreed, I only put it in there to trigger the discussion :)